app/services/google_parse_service.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class GoogleParseService
```

Comment:
We can add spec

Reply:
Done! Nice catch!
Fixed in [`5f004ca` (#47)](https://github.com/suho/google-search-ruby/pull/47/commits/5f004cae025caf7b86ce910239bd5478850fb86a)